private/buf/buflsp/symbol.go

```diff
@@ -523,6 +511,41 @@ func (s *symbol) FormatDocs(ctx context.Context) string {
 	return tooltip.String()
 }
 
+// formatComment takes a raw comment string and formats it by removing comment
+// delimiters and unnecessary whitespace. It handles both single-line (//) and
+// multi-line (/* */) comments.
+func formatComment(comment string) string {
```

Comment:
I think a better name for this is to rename it to something like commentToMarkdown, since the important thing is that this is emitting a Markdown string.